06_puppy/n0npax/types.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"sync"
+)
+
+type Puppy struct {
+	ID     int    `json:"id"`
+	Value  string `json:"value"`
+	Breed  string `json:"breed"`
+	Colour string `json:"colour"`
+}
+
+type Storer interface {
+	ReadPuppy(ID int) (*Puppy, error)
```

Comment:
Minor little nit. But the naming of input parameters here is inconsistent. We have `ID` and `id` across the different interface functions. Pick one and use that consistently.

Reply:
Fixed. All interfaces have capitalized `ID` now. Thanks!